views/flix.ejs

```diff
@@ -1,32 +1,54 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
     <link rel="stylesheet" href="css/style.css">
 </head>
+
 <body>
     <h1>FindAFlick</h1>
     <ul>
-    <% flix.forEach( el => { %>
+        <% flix.forEach( el=> { %>
             <li class='flixItem' data-id='<%=el._id%>'>
-                <span class='<%= el.completed === true ? 'completed' : 'not'%>'><%= el.flix %></span>
+                <span class='<%= el.completed === true ? ' completed' : 'not' %>'><%= el.flix %></span>
                 <span class='del'> Delete </span>
             </li>
-    <% }) %>    
+            <% }) %>
     </ul>
 
-    <h2><%= user.userName %> has <%= left %> movies & TV shows to watch.</h2>
-
-    <form action="/flix/createFlix" method='POST'>
-        <input type="text" placeholder="Enter Movie/Tv Show" name='flixItem'>
-        <input type="submit">
-    </form>
+    <input id='text-movie' type="text" placeholder="Enter Movie/Tv Show" name='flixItem'>
+    <button id='search-movie'>Submit</button>
```

Comment:
Can we update `Submit` to `Search`?

Reply:
Addressed in [d91369c](https://github.com/CadmiumRed40/FindAFlick/pull/24/commits/d91369cfcd3f96f18716d5d66f27145e258641cd).